app/Model/CoSetting.php

```diff
@@ -474,7 +480,21 @@ public function getThemeId($coId) {
   public function getTAndCLoginMode($coId) {
     return $this->lookupValue($coId, 't_and_c_login_mode');
   }
-  
+
+  /**
+   * Determine if Non-Admin Users can create Groups for the specified CO.
+   *
+   * @since  COmanage Registry v4.1.0
+   * @param  integer $coId CO ID
+   * @return boolean True if enabled, false otherwise
+   */
+
+  public function nonAdminsGroupCreate($coId) {
```

Comment:
We could be a bit wordier/clearer with this function name. Maybe something like `allowGroupCreationByNonAdmins()`?